picamera2/previews/drm_preview.py

```diff
@@ -49,13 +48,17 @@ class DrmPreview(NullPreview):
         "XBGR8888": pykms.PixelFormat.XBGR8888,
         "YUV420": pykms.PixelFormat.YUV420,
         "YVU420": pykms.PixelFormat.YVU420,
+        "MJPEG": pykms.PixelFormat.XBGR8888,
```

Comment:
I think we can simplify things a little if we map MJPEG onto `pykms.PixelFormat.BGR888`. We must also make a change below....

Reply:
Thanks, that seems a good idea